tests/Feature/BookTest.php

```diff
@@ -17,7 +17,8 @@ class BookTest extends TestCase
 
     public function test_get_all_books_success(): void
     {
-        Book::factory()->count(2)->create();
+        Book::factory(['claimed' => 0])->count(2)->create();
```

Comment:
Once you've done my comments about changing the route to show all books by default, make sure you pdate this test.

You should also add two additional tests, one that tests with claimed=1 and one with claimed=0 (pretty much copy and pastes of this test with some small changes)